_data/simple-icons.json

```diff
@@ -5300,6 +5300,11 @@
             "hex": "764ABC",
             "source": "https://github.com/reactjs/redux/tree/master/logo"
         },
+        {
+            "title": "RedwoodJS",
+            "hex": "BC4424",
```

Comment:
The color used in the SVG is `#BF4722`. Looks to be the primary color used in the website also. Where did you take this one?